torch/cuda/amp/grad_scaler.py

```diff
@@ -170,7 +48,8 @@ def scale(self, outputs):
             return outputs * self._scale.to(device=outputs.device, non_blocking=True)
 
         # Invoke the more complex machinery only if we're treating multiple outputs.
-        stash: List[_MultiDeviceReplicator] = []  # holds a reference that can be overwritten by apply_scale
+        # holds a reference that can be overwritten by apply_scale
+        stash: List[torch.amp.grad_scaler._MultiDeviceReplicator] = []
```

Comment:
Instead of typing out the full `torch.amp.grad_scaler.` every time in the file, can these be imported at the top for clarity + ease of reading?